run-tests.php

```diff
@@ -3650,9 +3648,9 @@ public function getExtensions(string $php): array
 class RuntestsValgrind
 {
     protected $version = '';
-    protected $header = '';
+    protected string $header = '';
     protected $version_3_8_0 = false;
-    protected $tool = null;
+    protected string $tool = '';
```

Comment:
Okay, that's what I did previously, I added the conversion just in case, as the signature of `preg_replace` and `version_compare` can also return other types. Thanks.

Reply:
Okay, it was probably my English, it's not in tune at all.

Adjustments sent.